src/common/functions.rs

```diff
@@ -22,7 +22,7 @@ pub fn init_vrl_runtime() -> vrl::compiler::runtime::Runtime {
 }
 
 pub fn get_vrl_compiler_config(org_id: &str) -> VRLCompilerConfig {
-    let lookup_tables = LOOKUP_TABLES.clone();
+    let lookup_tables = &LOOKUP_TABLES;
```

Comment:
This place needs careful testing, maybe it will cause deadlock. we got this issue earlier. because `LOOKUP_TABLES` is a RwMutex of HashMap.